### PR TITLE
fix: removed react aria from slider

### PR DESCRIPTION
**Pull request by cscheffauer**
# Description

- Had to remove react-aria from the slider cause of several dependency issue when consuming it.
- Switched the slider to native html element `<input type="range" .../>` and styled it accordingly.

### CODE REVIEW

File: src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -4,36 +4,35 @@ import classnames from 'classnames';
...
+      type="range"
+      value={value}
+      min={minValue}
+      max={maxValue}
+      aria-label={ariaLabel}
+      onChange={handleChange}
+      step={step}
+      className={classnames(className, STYLE.wrapper)}
+      id={id}
+      style={style}
```
Comment (nataliadelmar) [inline src/components/Slider/Slider.tsx:33]: does this make any difference if you are setting classname to STYLE.wrapper already?

File: src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -4,36 +4,35 @@ import classnames from 'classnames';
...
+      disabled={isDisabled}
```
Comment (nataliadelmar) [inline src/components/Slider/Slider.tsx:34]: disabled should have a default value of false

File: src/components/Slider/Slider.tsx [hunk 3]
```diff
@@ -4,36 +4,35 @@ import classnames from 'classnames';
...
-    </div>
+    <input
+      ref={inputRef}
+      type="range"
+      value={value}
+      min={minValue}
+      max={maxValue}
+      aria-label={ariaLabel}
+      onChange={handleChange}
+      step={step}
```
Comment (nataliadelmar) [inline src/components/Slider/Slider.tsx:30]: step, value, min and max should prob have default values as well

File: src/components/Slider/Slider.tsx [hunk 4]
```diff
@@ -4,36 +4,35 @@ import classnames from 'classnames';
...
-      </div>
-    </div>
+    <input
+      ref={inputRef}
+      type="range"
+      value={value}
+      min={minValue}
+      max={maxValue}
+      aria-label={ariaLabel}
+      onChange={handleChange}
```
Comment (nataliadelmar) [inline src/components/Slider/Slider.tsx:29]: types says that onChange is mandatory to be passed. Is it?

File: src/components/Slider/Slider.stories.tsx [hunk 5]
```diff
@@ -41,6 +41,7 @@ Example.args = {
   maxValue: 100,
   ariaLabel: 'slider',
   value: 0,
+  isDisabled: false,
```
Comment (nataliadelmar) [inline src/components/Slider/Slider.stories.tsx:44]: so we are assuming that the value will be controlled from outside, is that right? can storybook args pass functions? so, in that input box, can I put say '(value) => {console.log(value)}' and that would be rendered every time handleChange is called?

File: src/components/Slider/Slider.tsx (hunk 1 above)
Comment (cscheffauer) [reply src/components/Slider/Slider.tsx:33]: this is the inline style, which can be passed in by the consumer. Different then the STYLE.wrapper, which is a css `class`

File: src/components/Slider/Slider.stories.tsx (hunk 5 above)
Comment (cscheffauer) [reply src/components/Slider/Slider.stories.tsx:44]: yes, the slider is supposed to be a controlled component.
Check the stories file, i have done a wrapper component, which handles exactly this.
In storybook itself u will see "onChange" in the actions tab (next to the controls)

**Review by nataliadelmar — APPROVED** [review]

**Pull request merged**